integration/tests/postgres/multi-schema-support/specs/test-users.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: schemas.schemahero.io/v1alpha4
+kind: Table
+metadata:
+  name: test-users
+spec:
+  database: schemahero
+  name: users
+  schema: test
```

Comment:
this key is specific to postgres, and should be under the postgres: object in the spec.